### PR TITLE
Fix and update a few things in the documentation

### PR DESCRIPTION
**Pull request by bernhardmgruber**
I just gave the first documentation pages a quick look and corrected a few things.

### CODE REVIEW

File: docs/source/basic/cheatsheet.rst [hunk 1]
```diff
@@ -40,6 +40,7 @@ Define accelerator type (CUDA, OpenMP,etc.)
      .. code-block:: c++
 
 	AccGpuCudaRt,
+	AccGpuHipRt,
```
Comment (j-stephan) [inline docs/source/basic/cheatsheet.rst:43]: What about SYCL?

File: docs/source/basic/example.rst [hunk 2]
```diff
@@ -27,33 +27,25 @@ The following example shows a minimal example of a ``CMakeLists.txt`` that uses
...
+   project("myexample" CXX)
 
    find_package(alpaka REQUIRED)
-
-   alpaka_add_executable(${_TARGET_NAME} helloWorld.cpp)
-   target_link_libraries(
-     ${_TARGET_NAME}
-     PUBLIC alpaka::alpaka)
+   alpaka_add_executable(${CMAKE_PROJECT_NAME} helloWorld.cpp)
+   target_link_libraries(${CMAKE_PROJECT_NAME} PUBLIC alpaka::alpaka)
```
Comment (j-stephan) [inline docs/source/basic/example.rst]:
```suggestion
   alpaka_add_executable(${PROJECT_NAME} helloWorld.cpp)
   target_link_libraries(${PROJECT_NAME} PUBLIC alpaka::alpaka)
```

`CMAKE_PROJECT_NAME` always refers to the top-level project name.

File: docs/source/basic/example.rst [hunk 3]
```diff
@@ -64,15 +56,10 @@ The ``add_subdirectory`` method does not require alpaka to be installed. Instead
...
+   project("myexample" CXX)
 
    add_subdirectory(thirdParty/alpaka)
-
-   alpaka_add_executable(${_TARGET_NAME} helloWorld.cpp)
-   target_link_libraries(
-     ${_TARGET_NAME}
-     PUBLIC alpaka::alpaka)
+   alpaka_add_executable(${CMAKE_PROJECT_NAME} helloWorld.cpp)
+   target_link_libraries(${CMAKE_PROJECT_NAME} PUBLIC alpaka::alpaka)
```
Comment (j-stephan) [inline docs/source/basic/example.rst]:
```suggestion
   alpaka_add_executable(${PROJECT_NAME} helloWorld.cpp)
   target_link_libraries(${PROJECT_NAME} PUBLIC alpaka::alpaka)
```

File: docs/source/basic/library.rst [hunk 4]
```diff
@@ -117,7 +117,7 @@ Kernels can also be defined via lambda expressions.
    }
 
 .. attention::
-   The Nvidia ``nvcc`` does not support generic lambdas which are marked with `__device__`, which is what `ALPAKA_FN_ACC` expands to (among others) when the CUDA backend is active.
+   Nvidia's ``nvcc`` compiler does not support generic lambdas which are marked with `__device__`, which is what `ALPAKA_FN_ACC` expands to (among others) when the CUDA backend is active.
```
Comment (j-stephan) [inline docs/source/basic/library.rst]:
```suggestion
   NVIDIA's ``nvcc`` compiler does not support generic lambdas which are marked with `__device__`, which is what `ALPAKA_FN_ACC` expands to (among others) when the CUDA backend is active.
```

**Review by j-stephan — CHANGES_REQUESTED** [review]

File: docs/source/basic/cheatsheet.rst (hunk 1 above)
Comment (bernhardmgruber) [reply docs/source/basic/cheatsheet.rst:43]: Sure, tell me which names to put there! I wasn't sure myself.

Comment (j-stephan) [reply docs/source/basic/cheatsheet.rst:43]: alpaka provides the following concrete SYCL targets:

```c++
AccCpuSycl,
AccFpgaSyclIntel,
AccGpuSyclIntel
```

Comment (bernhardmgruber) [reply docs/source/basic/cheatsheet.rst:43]: Thx!

File: docs/source/basic/cheatsheet.rst [hunk 5]
```diff
@@ -40,6 +40,10 @@ Define accelerator type (CUDA, OpenMP,etc.)
...
+	AccCpuSycl,
+	AccFpgaSyclIntel,
+	AccGpuSyclIntel,
 	AccCpuOmp2Blocks,
 	AccCpuOmp2Threads,
 	AccCpuOmp4,
```
Comment (j-stephan) [inline docs/source/basic/cheatsheet.rst]: Just noticed `AccCpuOmp4`, it should be removed.

**Review by j-stephan — CHANGES_REQUESTED** [review]
Almost there, just one last thing:

**Review by j-stephan — APPROVED** [review]
Thank you for your service.

**Pull request merged**